README.rst

```diff
@@ -46,7 +46,9 @@ Usage Example
     # Configure I2C mode
```

Comment:
It would be good to keep this example as simple as possible for the most common usage.